src/api/app/models/project.rb

```diff
@@ -968,6 +976,44 @@ def branch_to_repositories_from(project, pkg_to_enable, opts = {})
     end
   end
 
+  def branch_remote_repositories(project)
```

Comment:
Here is the card:

https://trello.com/c/G3MNYknw/321-extract-branch-remote-repositories-functionality-from-the-project-model